charts/topolvm/values.yaml

```diff
@@ -290,16 +300,6 @@ node:
       #   cpu: "200m"
       #   memory: "200Mi"
 
-  # node.resources -- Specify resources.
-  ## ref: https://kubernetes.io/docs/user-guide/compute-resources/
-  resources: {}
```

Comment:
We think adding the `containers` field looks wordy, would you assemble all container resources under this section as follows?
```
resources:
  topolvm_node:
    requests:
     ...
  csi_registrar
```

Reply:
Done 62f221d08bed2121f9b576fef7ff17fb7b0fcbea